content/4_Gold/Combinatorics.problems.json

```diff
@@ -90,6 +90,71 @@
         "kind": "autogen-label-from-site",
         "site": "CF"
       }
+    },
+    {
+      "uniqueId": "usaco-862",
+      "name": "Cowpatibility",
```

Comment:
i wouldnt say this is a combo problem theres barely any involved, its just bitset optimization. why is it here?

Reply:
that's not the intended solution
intended solution uses PIE

Reply:
ok sure